src/lib/ilp-errors.ts

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+import * as IlpPacket from 'ilp-packet'
+
+export interface RejectionInfo {
+  code: string,
+  message: string,
+  data: Buffer
+}
+
+export const createIlpRejection = (account: string, { code, message, data }: RejectionInfo) => {
```

Comment:
Nit: technically this should be called `createIlpReject`

Reply:
Oops, that was actually dead code. Removed.